### PR TITLE
[OSCD] Always Install Elevated - Slide 50 - Rule 1

### PR DESCRIPTION
**Pull request by tas-kmanager**
Page [50](https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-50-638.jpg) from #574  Rule 1

Look for msiexec spawning command line or powershell

### CODE REVIEW

**Comment by tas-kmanager** [comment]
The test-sigma failed loading the dependencies, definitely need to re-run the test 😐

File: rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Image|contains: 
+            - '\cmd.exe'
+            - '\powershell.exe'
+    parent_image:
+        ParentImage|contains|all: 
+            - '\Windows\Installer\'
+            - 'msi'
+        ParentImage|endswith: 
+            - 'tmp'
+    condition: event_id and image and parent_image
```
Comment (w0rk3r) [inline rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell.yml:26]: Hey @tas-kmanager , nice contribution!

Here you reference an element that is not defined (event_id)

Comment (tas-kmanager) [reply rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell.yml]: Changes applied, thanks for the feedback!

**Comment by yugoslavskiy** [comment]
Thank you very much, @tas-kmanager!

File: rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell.yml (hunk 1 above)
Comment (yugoslavskiy) [reply rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell.yml]: Thank you for pointing this out, @w0rk3r!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**